parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -36,10 +36,16 @@ spec:
           mountPath: "/etc/kubernetes"
         - name: "var-lib-kubelet"
           mountPath: "/var/lib/kubelet"
+        - name: "msi"
+          mountPath: "/var/lib/waagent/ManagedIdentity-Settings"
+          readOnly: true
   volumes:
     - name: "etc-kubernetes"
       hostPath:
         path: "/etc/kubernetes"
     - name: "var-lib-kubelet"
       hostPath:
         path: "/var/lib/kubelet"
+    - name: msi
+      hostPath:
+      path: "/var/lib/waagent/ManagedIdentity-Settings"
```

Comment:
Good catch. What's weird is:
1. I don't know what apiserver uses cloudprovider for...
2. I'm surprised everything worked... I would've expected a problem if it tried to actually get a token for something...

Will fix. Looking into why this didn't surface any sort of failure to me...